src/ipc/handler.c

```diff
@@ -122,8 +122,8 @@ struct sof_ipc_cmd_hdr *mailbox_validate(void)
 	mailbox_hostbox_read(hdr, SOF_IPC_MSG_MAX_SIZE, 0, sizeof(*hdr));
 
 	/* validate component header */
-	if (hdr->size > SOF_IPC_MSG_MAX_SIZE) {
-		tr_err(&ipc_tr, "ipc: msg too big at 0x%x", hdr->size);
+	if (!hdr->size || hdr->size > SOF_IPC_MSG_MAX_SIZE) {
```

Comment:
Should the min size not be sizeof(header) as this will be our smallest message ?

Reply:
looks even better, done